build_sdk.yml

```diff
@@ -28,237 +32,16 @@ jobs:
-              const pythonInstallCommand = pythonRepo ? `# python
-            pip install git+https://github.com/${pythonRepo.owner}/${pythonRepo.name}.git@${pythonRepo.branch}` : '';
-
-              const installation = npmInstallCommand || pythonInstallCommand ? `#### :package: Installation
-            ${[npmInstallCommand, pythonInstallCommand].filter(Boolean).map(cmd => `\`\`\`bash\n${cmd}\n\`\`\``).join('\n')}` : '';
-
-              const commentBody = `
-            ### :sparkles: SDK Previews
-            _Last updated: ${new Date().toISOString().replace('T', ' ').replace(/\.\d+Z$/, ' UTC')}_
-
-            The following table summarizes the build outcomes for all languages:
-
-            ${header}${tableRows}
-
-            You can freely modify the branches to add [custom code](https://app.stainlessapi.com/docs/guides/patch-custom-code).${installation ? `\n${installation}` : ''}
-              `;
-
-              const { data: comments } = await github.rest.issues.listComments({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                issue_number: context.payload.pull_request.number,
-              });
-
-              const existingComment = comments.find((comment) =>
-                comment.body.includes("### :sparkles: SDK Previews")
-              );
-
-              if (existingComment) {
-                await github.rest.issues.updateComment({
-                  owner: context.repo.owner,
-                  repo: context.repo.repo,
-                  comment_id: existingComment.id,
-                  body: commentBody,
-                });
-              } else {
-                await github.rest.issues.createComment({
-                  owner: context.repo.owner,
-                  repo: context.repo.repo,
-                  issue_number: context.payload.pull_request.number,
-                  body: commentBody,
-                });
-              }
-
-      - name: Check results
-        if: steps.config-changed.outputs.changed == 'true'
-        run: |
-          FAIL_RUN_ON="${{ env.FAIL_RUN_ON }}"
-          [ "$FAIL_RUN_ON" = "never" ] && exit 0
-
-          FILTER=".value.commit == null"
-          [ "$FAIL_RUN_ON" = "error" -o "$FAIL_RUN_ON" = "warning" -o "$FAIL_RUN_ON" = "note" ] && FILTER="$FILTER or .value.conclusion == \"error\""
-          [ "$FAIL_RUN_ON" = "warning" -o "$FAIL_RUN_ON" = "note" ] && FILTER="$FILTER or .value.conclusion == \"warning\""
-          [ "$FAIL_RUN_ON" = "note" ] && FILTER="$FILTER or .value.conclusion == \"note\""
-
-          FAILED_LANGUAGES=$(echo '${{ steps.build-sdk.outputs.results }}' | jq -r ".outcomes | to_entries[] | select($FILTER) | .key")
-
-          if [ -n "$FAILED_LANGUAGES" ]; then
-            echo -e "\e[31mThe following languages did not build successfully: $FAILED_LANGUAGES\e[0m"
-            exit 1
-          fi
+          org_name: ${{ env.STAINLESS_ORG_NAME }}
```

Comment:
suggestion: I think we should do `project: ${{ }}` and `org: ${{ }}` everywhere that is user facing.